main.go

```diff
@@ -2,24 +2,40 @@
 package main
 
 import (
+	"flag"
+	"os"
+	"strings"
 	"student20_pop/WebSocket"
 	"text/template"
 
-	"flag"
 	"log"
 	"net/http"
 )
 
 // this function basically makes the webserver run
 func main() {
 
```

Comment:
careful, you're not validating any of the args.
You could make your life easy by using a cli dependency that allows you to define flags, acceptable formats, cli documentation, and what not.

Reply:
yep, I didn't know we could do this, thank you. It's implemented now :)